quartz_solar_forecast/forecast.py

```diff
@@ -30,6 +22,69 @@ def run_forecast(site: PVSite, ts: datetime | str = None, nwp_source: str = "ico
     pv_xr = make_pv_data(site=site, ts=ts)
 
     # load and run models
-    pred_df = forecast_v1(nwp_source, nwp_xr, pv_xr, ts)
+    pred_df = forecast_v1(nwp_source, nwp_xr, pv_xr, ts, model=model)
 
     return pred_df
+
+
+def predict_tryolabs(
+    site: PVSite, ts: datetime | str = None):
+    """Run the forecast with the tryolabs model"""
+
+    solar_power_predictor = TryolabsSolarPowerPredictor()
```

Comment:
could you add a few # comments, just saying what the code is doing, for example `setting up model`